packages/sources/svelte/src/components/structure/VtmnCard/VtmnCard.svelte

```diff
@@ -24,12 +24,20 @@
    */
   export let contentOpaque = false;
 
+  /**
+   * @type {1|2|3|4|5|6} Heading level of card title
+   * @default 2
+   */
+  export let headingLevel = 2;
```

Comment:
We have to keep the default behaviours. We already used the VtmnCard on the ecomm website.

```suggestion
  /**
   * @type {1|2|3|4|5|6} Heading level of card title
   */
  export let headingLevel;
```